Assets/BossRoom/Scripts/Client/Game/Action/DashAttackActionFX.cs

```diff
@@ -0,0 +1,98 @@
+using MLAPI;
```

Comment:
when I click to move the character left, hit the dash shortcut, then while ramping up I click to the left of the character I get a weird boomerang movement

Reply:
What's happening there is that the action is canceled due to movement. I've cleaned up the action-cancel code to explicitly cancel the FX